chef_master/source/install_server_tiered.rst

```diff
@@ -20,6 +20,22 @@ Before installing the Chef server software, perform the following steps:
 * The hostname for the Chef server must be an FQDN, including the domain suffix, and must be resolvable by the backend and frontend servers. See `Hostnames, FQDNs </install_server_pre.html#hostnames>`_ for more information.
 * ``chef-server-ctl reconfigure`` will not bind the ``backend_vip`` to the backend server. The easiest thing to do is just define ``backend_vip`` as the already configured main IP address of the backend system. If you need to use an additional address, it will need to be configured and bound on the system before ``chef-server-ctl reconfigure`` is run.
 
+Basic Hardware Requirements
+=====================================================
+For a tiered deployment, your backend server should support the following hardware requirements:
+
+* 64-bit architecture
+* 8 total cores (physical or virtual)
+* 16GB RAM
+* 2 x 300GB SAS RAID1 drives
```

Comment:
Why do we recommend 300GB of disk space in tier, but only 50GB of disk space in HA?